### PR TITLE
Initial Scripts

### PR DESCRIPTION
**Pull request by agorina**
Initial scripts. 
Note that it assumes MySQL 5.6 since this is the version supported by Amazon Aurora.

Both reporting and warehouse are identical for now.
It is enough to review just one.

### CODE REVIEW

File: reporting/sql/V201702061486427095__initial_dml.sql [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  (2, 'AmericanIndianOrAlaskaNative'),
+  (3, 'Asian'),
+  (4, 'BlackOrAfricanAmerican'),
+  (5, 'NativeHawaiianOrOtherPacificIslander'),
+  (6, 'DemographicRaceTwoOrMoreRaces');
+
+INSERT INTO gender (id, name) VALUES
+  (0, 'undefined'),
+  (1, 'male'),
+  (2, 'female');
```
Comment (malaffoon) [inline reporting/sql/V201702061486427095__initial_dml.sql]: (3, 'fluid'),
(4, 'FtM'),
(5, 'MtF')

;)

File: reporting/sql/V201702061486427077__initial_ddl.sql [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+);
+
+CREATE TABLE IF NOT EXISTS gender (
+  id tinyint NOT NULL PRIMARY KEY, 
+  name varchar(255) NOT NULL UNIQUE
+);
+
+CREATE TABLE IF NOT EXISTS accommodation (
+  id int NOT NULL AUTO_INCREMENT PRIMARY KEY, 
+  code varchar(25) NOT NULL UNIQUE
```
Comment (malaffoon) [inline reporting/sql/V201702061486427077__initial_ddl.sql]: just the code? description and value/type not needed?

Comment (agorina) [inline reporting/sql/V201702061486427077__initial_ddl.sql]: I do not know. I decided to address it later when I understand better what is needed. I have it on my TODO list.

File: reporting/sql/V201702061486427095__initial_dml.sql (hunk 1 above)
Comment (agorina) [inline reporting/sql/V201702061486427095__initial_dml.sql]: Will add.

**Comment by agorina** [comment]
Initial scripts.
Note that it assumes MySQL 5.6 since this is the version supported by Amazon Aurora.
Both reporting and warehouse are identical for now.

**Pull request closed without merging**